metadata.rb

```diff
@@ -4,7 +4,7 @@
 maintainer_email 'jj@chef.io'
 license 'Apache-2.0'
 description 'Installs and configures RabbitMQ server'
-version '5.1.0'
+version '5.1.1'
```

Comment:
Please revert this. It's up to the maintainers to decide what the next version should be (and why).